Code/GraphMol/Fingerprints/RDKitFPGenerator.cpp

```diff
@@ -90,21 +90,32 @@ RDKitFPArguments<OutputType>::RDKitFPArguments(
 
 template <typename OutputType>
 OutputType RDKitFPAtomEnv<OutputType>::getBitId(
-    FingerprintArguments<OutputType> *,  // arguments
-    const std::vector<std::uint32_t> *,  // atomInvariants
-    const std::vector<std::uint32_t> *,  // bondInvariants
-    const AdditionalOutput *,            // additionalOutput
-    const bool                           // hashResults
-) const {
-  // todo set additional outputs
+    FingerprintArguments<OutputType> *,                    // arguments
+    const std::vector<std::uint32_t> *,                    // atomInvariants
+    const std::vector<std::uint32_t> *,                    // bondInvariants
+    const AdditionalOutput *additionalOutput, const bool,  // hashResults
+    const std::uint64_t fpSize) const {
+  if (additionalOutput) {
+    OutputType bit_id = fpSize ? (d_bitId % fpSize) : d_bitId;
```

Comment:
The ternary operator is not too readable... I generally prefer to use and `if` statement. Maybe:
```suggestion
    OutputType bit_id = d_bitId;
    if (fpSize) {
        bit_id %= fpSize;
    }
```